src/workflows/context/context.py

```diff
@@ -414,7 +423,7 @@ def _get_event_buffer_id(self, events: list[Type[Event]]) -> str:
         # Fall back to creating a stable identifier from expected events
         return ":".join(sorted(self._get_full_path(e_type) for e_type in events))
 
-    def collect_events(
```

Comment:
Let's keep this around and alias it to `gather_events` with a deprecation notice